README.md

```diff
@@ -115,13 +149,26 @@ any given point in time, fully support a maximum of 2 LTS releases.
 [`NAN`]: https://github.com/nodejs/nan
 
 ## LTS Team members
-
 * Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
-* James M Snell [@jasnell](https://github.com/jasnell)
-* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123)
+* James M Snell [@jasnell](https://github.com/jasnell) 
+* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123) 
 * Michael Dawson [@mhdawson](https://github.com/mhdawson)
-* Myles Borins [@MylesBorins](https://github.com/MylesBorins)
-* Rod Vagg [@rvagg](https://github.com/rvagg)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins) 
 * Sam Roberts [@sam-github](https://github.com/sam-github)
 
-Github team for LTS: https://github.com/orgs/nodejs/teams/lts
+### Releasers team
+* Italo A. Casas [@italoacasas](https://github.com/italoacasas) (release team)
+* James M Snell [@jasnell](https://github.com/jasnell) (release team)
+* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123) (release team)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins) (release team)
+* Rod Vagg [@rvagg](https://github.com/rvagg) (release team)
+
+### CITGM team
+* James M Snell [@jasnell](https://github.com/jasnell) (release team)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins) (release team)
+* Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
+* George Adams [George Adams](https://github.com/gdams)
```

Comment:
I meant that starting with George, all of the CITGM members have their name as the link, not their GitHub username. So, this line specifically should probably be `* George Adams [@gdams](https://github.com/gdams)` and the ones below should follow that same format.